docker-compose.yml

```diff
@@ -10,12 +10,13 @@ services:
      - zookeeper
      - kafka
     environment:
-      NAKADI_OAUTH2_MODE: "OFF"
-      NAKADI_FEATURETOGGLE_DEFAULT: "true"
-      NAKADI_ZOOKEEPER_BROKERS: localhost:2181
-      SPRING_DATASOURCE_URL: jdbc:postgresql://localhost:5432/local_nakadi_db
-      SPRING_DATASOURCE_USERNAME: nakadi
-      SPRING_DATASOURCE_PASSWORD: nakadi
+      - NAKADI_STREAM_DEFAULT_COMMITTIMEOUT
```

Comment:
It's a feature from docker compose. It just transport this environment variable from the shell, if defined, into the docker container.

The default value is in the application.yml from the application.

One needs to keep in mind the requirements: (1) 5 seconds for tests, (2) 60 seconds for local execution.